related-tooling.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Related Tooling
+sidebar: home_sidebar
+permalink: /related-tooling/
+toc: false
+---
+
+Open Source tooling, such as for managing the state of Pulp and integrating it with other projects, that is maintained by the commnunity or other projects.
+
+## Foreman / Katello
+
+[Foreman](https://theforeman.org/) is a complete lifecycle management tool for physical and virtual servers. [Katello](https://theforeman.org/plugins/katello/) brings the full power of content management alongside the provisioning and configuration capabilities of Foreman.
+
+Foreman & Katello use Pulp for managing repositories software packages. It provides a graphical WebUI for Pulp and integrates it with its workflows, but does not expose every underlying feature & content type of Pulp.
+
+Foreman & Katello are currently in the incremental [process of migrating from Pulp 2 to Pulp 3.](https://community.theforeman.org/t/update-on-katello-pulp3-integration-and-upcoming-roadmap/15899)
+
+## Ansible Modules for Pulp aka Squeezer
+
+[This project](https://github.com/mdellweg/ansible_modules_pulp) project aims to provide a complete set of ansible modules to control a pulp3 server in a descriptive way. This is neither to be confused with ansible-pulp to install pulp, nor pulp_ansible to manage ansible content in pulp. It has been wildly inspired by the sister project [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules).
```

Comment:
:champagne:
I know, i wrote this text, but i think, we should leave the aim to the project page.
Here we could skip the "complete" fore now and add a note that file related modules are already available.

Reply:
Updated, thanks!